site/student-lectures/Differential_Evolution/Differential_Evolution.rst

```diff
@@ -0,0 +1,140 @@
+*******************************************
+Introduction to Differential Evolution (DE)
+*******************************************
+* DE is a method in evolutionary computation that optimizes solutions to problem by iteratively improving a candidate solution based on a quality measure.
+* It's a type of metaheuristic, making few assumptions about the problem, enabling exploration of large solution spaces.
+* DE works well for multidimensional real-valued functions, not requiring differentiability, allowing application to non-continuous, noisy, or time-varying problem.
+
+    .. figure:: Differential_evolution_flow_chart.png
+        :width: 224 px
+        :align: center
+
+        Figure 1: Differential Evolution (DE) is a type of evolutionary algorithm that optimizes a problem by iteratively improving a candidate solution with regard to a given measure of quality.
+            Image credit: `Xiu Zhang <https://www.researchgate.net/publication/310811940_Shift_based_adaptive_differential_evolution_for_PID_controller_designs_using_swarm_intelligence_algorithm>`_
+
+********************************
+Algorithm Overview and Mechanics
+********************************
```

Comment:
See #254 

You need to change these to sections. You currently have them such that they are like new topics.

Change to this:

```
Algorithm Overview and Mechanics
===========================
```